### DISCUSSION TITLE
Is there a way to save all the inputs each session?

### DISCUSSION THREAD

**Discussion by Luna447** [TheLastBen/fast-stable-diffusion#189] [Q&A] [answered]
I'm using the AUTOMATIC1111 UI and especially the Batch img2img function to generate more than 3k images for a video.
Is there a way to save and import all the settings like current promt, negative prompt, in and output dir, steps, width, height etc. ?

**Comment by TheLastBen** [comment, marked as answer]
for such issue, refer to this repo for better answers : https://github.com/AUTOMATIC1111/stable-diffusion-webui/

**Reply by Luna447** [reply]
[If someone would like to join the discussion](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/3229) <-
